js/ui/grid_core/ui.grid_core.grid_view.js

```diff
@@ -160,19 +162,28 @@ var ResizingController = modules.ViewController.inherit({
     _toggleBestFitModeForView: function(view, className, isBestFit) {
         if(!view || !view.isVisible()) return;
 
-        var $rowsTable = this._rowsView._getTableElement(),
-            $viewTable = view._getTableElement(),
+        var $rowsTable,
+            $viewTable,
+            isFixedTable,
+            $rowsTables = this._rowsView.getTableElements(),
+            $viewTables = view.getTableElements(),
             $tableBody;
 
-        if($viewTable) {
-            if(isBestFit) {
-                $tableBody = $viewTable.children("tbody").appendTo($rowsTable);
-            } else {
-                $tableBody = $rowsTable.children("." + className).appendTo($viewTable);
+        each($rowsTables, (_, tableElement) => {
+            $rowsTable = $(tableElement);
+            isFixedTable = $rowsTable.hasClass(POINTER_EVENTS_NONE_CLASS);
+            $viewTable = $viewTables.filter(isFixedTable ? `.${POINTER_EVENTS_NONE_CLASS}` : `:not(.${POINTER_EVENTS_NONE_CLASS})`);
```

Comment:
Maybe move the inner filter condition to the local var to avoid repeated additional checks?

Reply:
May be get correct $viewTable by index:
```
each($rowsTables, (index, tableElement) => {
    ...
    $viewTable = $viewTables.eq(index);
```